cookbooks/ey-newrelic_infra/files/default/newrelic-infra.list

```diff
@@ -0,0 +1 @@
+deb [arch=amd64] https://download.newrelic.com/infrastructure_agent/linux/apt focal main
```

Comment:
This file should be generated when adding also, the type should support arm64 as well as amd64

Reply:
Removed files folder as `newrelic-infra.list` should be generated upon execution of `apt_repository` process